lte/gateway/python/magma/pipelined/openflow/registers.py

```diff
@@ -17,6 +17,19 @@
 PASSTHROUGH_REG = 'reg6'
 VLAN_TAG_REG = 'reg7'
 
+# xxreg3 allow us to specify 16 bytes vakue to describe APN
+# according to http://man7.org/linux/man-pages/man7/ovs-fields.7.html 
+# xxreg3 will allocate [xreg2, xreg3] or [reg12, reg13, reg14, reg15] 
+# under the hood
+APN_TAG_REG = 'xxreg3' 
```

Comment:
does this belong in apn.py instead (will anyone else use it)? We should probably move the most of the global registered defined above out of here eventually for the same reason